### PR TITLE
Implement GUI tab switch when typing in view commands

### LINKED ISSUE #204: Implement tab switch when user inputs view commands
As a user i want to view the relevant information when a view command is inputted.

### PR DESCRIPTION
**Pull request by ngzhenteng**
Consultation tab is not complete yet, however when it is fully implemented the tab switch should function as expected.
Fixes #204

### CODE REVIEW

**Comment by wireseo** [comment]
Nice work! And sorry for the consultation tab -- I'll try to make it work asap :0

File: src/main/java/seedu/address/logic/commands/ViewCommandType.java [hunk 1]
```diff
@@ -0,0 +1,13 @@
+package seedu.address.logic.commands;
```
Comment (Asuraxsoul) [inline src/main/java/seedu/address/logic/commands/ViewCommandType.java]: good use of enum

**Review by Asuraxsoul — COMMENTED** [review]
Great GUI tab switching implementation

**Review by peironggg — COMMENTED** [review]
Amazing job, def not an easy feature to pull off.

**Comment by wilinetan** [comment]
Thanks for implementing this feature!

**Pull request merged**